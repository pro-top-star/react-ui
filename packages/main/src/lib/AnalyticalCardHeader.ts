import { AnalyticalCardHeader } from '../components/AnalyticalCard/header/AnalyticalCardHeader';

export { AnalyticalCardHeader };
