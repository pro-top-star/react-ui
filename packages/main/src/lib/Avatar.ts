import { Avatar } from '../webComponents/Avatar';

export { Avatar };
