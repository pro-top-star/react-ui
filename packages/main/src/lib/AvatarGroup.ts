import { AvatarGroup } from '../webComponents/AvatarGroup';
import type { AvatarGroupPropTypes } from '../webComponents/AvatarGroup';

export { AvatarGroup };
export type { AvatarGroupPropTypes };
