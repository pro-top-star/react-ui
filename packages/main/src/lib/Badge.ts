import { Badge } from '../webComponents/Badge';
import type { BadgePropTypes } from '../webComponents/Badge';

export { Badge };
export type { BadgePropTypes };
