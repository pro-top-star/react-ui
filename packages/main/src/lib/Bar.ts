import { Bar } from '../webComponents/Bar';
import type { BarPropTypes } from '../webComponents/Bar';

export { Bar };
export type { BarPropTypes };
