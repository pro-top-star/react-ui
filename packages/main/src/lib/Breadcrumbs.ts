import { Breadcrumbs } from '../components/Breadcrumbs';

export { Breadcrumbs };
