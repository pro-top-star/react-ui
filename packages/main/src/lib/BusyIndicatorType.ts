import { BusyIndicatorType } from '../enums/BusyIndicatorType';

export { BusyIndicatorType };
