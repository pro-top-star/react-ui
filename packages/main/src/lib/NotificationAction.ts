import { NotificationAction } from '../webComponents/NotificationAction';
import type { NotificationActionPropTypes } from '../webComponents/NotificationAction';

export { NotificationAction };
export type { NotificationActionPropTypes };
