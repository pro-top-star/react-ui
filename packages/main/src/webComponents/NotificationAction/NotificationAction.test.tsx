import { render } from '@shared/tests';
import { NotificationAction } from '@ui5/webcomponents-react/lib/NotificationAction';
import React from 'react';

describe('NotificationAction', () => {
  test('Basic Test (generated)', () => {
    const { asFragment } = render(<NotificationAction />);
    expect(asFragment()).toMatchSnapshot();
  });
});
