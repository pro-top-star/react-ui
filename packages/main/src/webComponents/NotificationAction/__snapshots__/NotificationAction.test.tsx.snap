// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotificationAction Basic Test (generated) 1`] = `
<DocumentFragment>
  <ui5-notification-action
    design="Transparent"
    ui5-notification-action=""
  />
</DocumentFragment>
`;
